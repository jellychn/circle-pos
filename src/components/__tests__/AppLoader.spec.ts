import { shallowMount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'
import AppLoader from '../AppLoader.vue'

describe('AppLoader.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(AppLoader)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
