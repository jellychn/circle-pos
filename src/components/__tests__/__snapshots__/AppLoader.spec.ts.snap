// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppLoader.vue > matches snapshot 1`] = `"<div data-v-8ec46d9a="" class="loader"><span data-v-8ec46d9a="" class="spinner"></span></div>"`;
